src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -207,8 +207,8 @@ export default class SimidMapCreative extends BaseSimidCreative {
     returnToAdButton.classList.add("hidden");
     const mapDiv = document.getElementById("map");
     mapDiv.classList.add("hidden");
-    const travelDisplay = document.getElementById("travel_method");
-    const timeDisplay = document.getElementById("time_display");
+    const travelDisplay = document.getElementById("travelMethod");
```

Comment:
These are being created dynamically right? So the id is also set in javascript, not in html?  If thats true, you can replace these with a constant and avoid this kind of error.

Reply:
Done!